encoding not installed by node-fetch

warn ../node_modules/.pnpm/node-fetch@2.6.11/node_modules/node-fetch/lib/index.js
Module not found: Can't resolve 'encoding' in '/__w/plan-prise-next/plan-prise-next/node_modules/.pnpm/node-fetch@2.6.11/node_modules/node-fetch/lib'

Assignees: 
Labels: 



<!-- Edit the body of your new issue then click the ✓ "Create Issue" button in the top right of the editor. The first line will be the issue title. Assignees and Labels follow after a blank line. Leave an empty line before beginning the body of the issue. -->

--- a/.pnpmfile.cjs
+++ b/.pnpmfile.cjs
@@ -1,0 +1,10 @@
+module.exports = {
+  hooks: {
+    readPackage: (pkg) => {
+      if (pkg.name === "node-fetch") {
+        pkg.dependencies["encoding"] = "0.1.13";
+      }
+      return pkg;
+    },
+  },
+};

